pages/test_analytics/ci_environments.md.erb

```diff
@@ -0,0 +1,55 @@
+# CI Environments
+
+Buildkite Test Analytics collectors automatically detect common continuous integration (CI) environments.
+If available, test collectors gather information about your test runs, such as branch names and build IDs.
+Test collectors gather information from the following CI environments:
+
+- [Buildkite](#buildkite)
+- [CircleCI](#circleci)
+- [GitHub Actions](#github-actions)
+
+If you use a test collector in any other environment, the test run will be assigned a unique identifier and attributed to a "generic" CI system.
+Other information, such as branch names or commit hashes, will not be collected.
+
+If you're not using a test collector, see [Importing JSON](/docs/test-analytics/importing-json) and [Importing JUnit XML](/docs/test-analytics/importing-junit-xml) to learn how to provide run environment data.
+
+{:toc}
+
+## Buildkite
```

Comment:
For each of the CI provider sections, I've tried to adopt the terminology that gets used by the CI provider, rather than Buildkite's, providing a little bit of a bridge between various terminologies.

Reply:
:+1: